plugins/inputs/aerospike/aerospike.go

```diff
@@ -436,13 +455,78 @@ func parseAerospikeValue(key string, v string) interface{} {
 		return parsed
 	} else if parsed, err := strconv.ParseBool(v); err == nil {
 		return parsed
+	} else if parsed, err := strconv.ParseFloat(v, 64); err == nil {
+		return parsed
 	} else if parsed, err := strconv.ParseFloat(v, 32); err == nil {
 		return parsed
 	}
 	// leave as string
 	return v
 }
 
+func (a *Aerospike) parseLatencyInfo(acc telegraf.Accumulator, latencyInfo, hostPort, nodeName string) {
+	latencies := strings.Split(latencyInfo, ";")
+	i := 0
+	for i < len(latencies) {
+		latency := latencies[i]
+		if strings.Compare(latency, "error-no-data-yet-or-back-too-small") == 0 {
+			i++
+		} else {
```

Comment:
Could maybe add a `continue` here to reduce the indentation a bit?

Reply:
Added.